manim/utils/config_ops.py

```diff
@@ -29,6 +30,13 @@ def merge_dicts_recursively(*dicts):
     return result
 
 
+def update_dict_recursively(*dicts):
+    current_dict = dicts[0]
+    updated_dict = merge_dicts_recursively(*dicts)
+    for key, value in updated_dict.items():
+        current_dict[key] = value
```

Comment:
The current code can error on `dicts[0]` if no dicts are passed; better to be explicit in the signature about how many arguments are needed imo. Also the for loop can just be replaced with a call to `dict.update` afaict
```suggestion
def update_dict_recursively(current_dict, *others):
    updated_dict = merge_dicts_recursively(current_dict, *others)
    current_dict.update(updated_dict)
```

Reply:
Ah, didn't think of that, that's even better, thanks!